.github/workflows/release-workflow-dispatcher.yml

```diff
@@ -0,0 +1,284 @@
+        run: |
+          nix-shell --pure --run "./scripts/helm/test-update-chart-version.sh" ./shell.nix
+
+      - name: Modify the chart version based on the branch name for develop
+        run: |
+          branch_name=${{ github.ref_name }}
+          nix-shell --pure --run "./scripts/helm/update-chart-version.sh --branch $branch_name --type develop --localpv-provisioner-version ${LOCALPV_PROVISIONER_VERSION} --zfs-localpv-version ${ZFS_LOCALPV_VERSION} --lvm-localpv-version ${LVM_LOCALPV_VERSION} --mayastor-version ${MAYASTOR_VERSION}" ./shell.nix
+        env:
+          LOCALPV_PROVISIONER_VERSION: ${{ steps.extract-versions.outputs.localpv_provisioner_version }}
+          ZFS_LOCALPV_VERSION: ${{ steps.extract-versions.outputs.zfs_localpv_version }}
+          LVM_LOCALPV_VERSION: ${{ steps.extract-versions.outputs.lvm_localpv_version }}
+          MAYASTOR_VERSION: ${{ steps.extract-versions.outputs.mayastor_version }}
+
+      - name: Create Pull Request to develop
+        id: cpr
+        uses: peter-evans/create-pull-request@v5
+        with:
+          base: develop
+          commit-message: "chore(ci): update helm chart versions and/or git submodules"
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          title: "Prepare develop branch on ${{ github.ref_name }} creation"
+          labels: |
+            update-develop-branch
+            automated-pr
+          draft: false
+          signoff: true
+          branch: "create-pull-request/patch-develop"
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+  update_release_branch_after_release:
+    outputs:
+      localpv_provisioner_version: ${{ steps.extract-versions-from-tag.outputs.localpv_provisioner_version }}
+      zfs_localpv_version: ${{ steps.extract-versions-from-tag.outputs.zfs_localpv_version }}
+      lvm_localpv_version: ${{ steps.extract-versions-from-tag.outputs.lvm_localpv_version }}
+      mayastor_version: ${{ steps.extract-versions-from-tag.outputs.mayastor_version }}
+    runs-on: ubuntu-22.04
+    if: github.ref_type == 'tag' && github.event_name == 'push' && !contains(github.ref_name, 'rc')
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          #ref: ${{ github.ref_name }}
+          fetch-depth: 0
+          submodules: recursive
+
+      - uses: cachix/install-nix-action@v22
+      - name: Pre-populate nix-shell
+        run: |
+          export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' nix/sources.json -r)
+          echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
+          nix-shell --pure --run "echo" ./shell.nix
+
+      - name: Get dependencies version from release tag
+        id: extract-versions-from-tag
+        run: |
+          GITHUB_API_URL="https://api.github.com/repos"
+
+          # Define output file for GitHub Actions
+          OUTPUT_FILE="$GITHUB_OUTPUT"
+
+          # Fetch and extract versions
```

Comment:
Shouldn't we simply remove the -prerelease?
Otherwise this could point the release to an incorrect version. Ex: let's say we release a patch on a previous release, we can't just update the dependencies to their latest.

Reply:
Not sure, we remove the '-prerelease' when we create a release. Patch releases won't be marked as latest. Maybe we can discuss this ?

Reply:
Also , we can update the versions when the PR is created. It is not an automated merge considering. So once the PR is created it can always be manually edited.

Reply:
> Not sure, we remove the '-prerelease' when we create a release. Patch releases won't be marked as latest. Maybe we can discuss this ?

Yeah but the chart without -prerelease is not actually checked in, it's only pushed to the helm registry?

Reply:
Yes, thats correct. I just followed the design and workflow of other repos. It's the same behaviour across.